File: Codewars/1-evenOrOdd.js
// Codewars 1
// Title : Even or Odd
// Question : Create a function that takes an integer as an argument and returns "Even" for even numbers or "Odd" for odd numbers.
// Challange Url : https://www.codewars.com/kata/53da3dbb4a5168369a0000fe
// Level : 8kyu

// Method : 1
// function evenOrOdd(number) {
//     return number / 2 == 0 ? "Odd" : "Even";
// }

// Method : 2
const evenOrOdd = (number) => (number % 2 === 0 ? "Even" : "Odd");

// Other Users Method
// const evenOrOdd = (number) => (number % 2 ? "Even" : "Odd");

console.log(evenOrOdd(2));
console.log(evenOrOdd(7));
console.log(evenOrOdd(-42));
console.log(evenOrOdd(-7));
console.log(evenOrOdd(0));

File: Codewars/python/8-kyu/1-even_or_odd.py
# Codewars 1
# Title : Even or Odd
# Question : Create a function that takes an integer as an argument and returns "Even" for even numbers or "Odd" for odd numbers.
# Challange Url : https://www.codewars.com/kata/53da3dbb4a5168369a0000fe
# Level : 8kyu

# Method : 1
# def even_or_odd(number):
#     if number % 2 == 0:
#       return 'Odd' 
#     else:
#       return 'Even' 

# Method : 2
def even_or_odd(n) : return "Odd" if n % 2==0 else 'Even' 


# Other Users Method

# Log or debug
print(even_or_odd(2))
print(even_or_odd(4))
print(even_or_odd(7))
print(even_or_odd(9))
print(even_or_odd(23))

